tools/mavlogdump.py

```diff
@@ -76,6 +76,10 @@
     yappi.start()
 
 if args.format == 'mat':
+    # Check that the mat_file argument has been specified
+    if not args.mat_file:
```

Comment:
Should this be
```suggestion
    if not args.mat_file is not None:
```

... i.e. we don't care about the truthiness of the value, just that it was supplied.

Reply:
Changed to "if args.mat_file is None" (rather than double negative!)